flink-state-backends/flink-statebackend-changelog/src/main/java/org/apache/flink/state/changelog/AbstractStateChangeLogger.java

```diff
@@ -125,6 +152,9 @@ protected void log(
         return serializeRaw(
                 wrapper -> {
                     wrapper.writeByte(op.code);
+                    // todo: optimize in FLINK-22944 by either writing short code or grouping and
+                    // writing once (same for key, ns)
```

Comment:
I think two more spaces here is better, so that we can know this is anther line of above TODO.